harness/determined/deploy/local/docker-compose.yaml

```diff
@@ -25,6 +25,9 @@ services:
       DET_LOG_LEVEL: ${INTEGRATIONS_LOG_LEVEL:-info}
       DET_MASTER_HTTP_PORT: ${INTEGRATIONS_HOST_PORT:-8080}
       DET_DB_PASSWORD: ${DET_DB_PASSWORD}
+      DET_SCIM_ENABLED: ${DET_SCIM_ENABLED}
+      DET_SCIM_USERNAME: ${DET_SCIM_USERNAME}
+      DET_SCIM_PASSWORD: ${DET_SCIM_PASSWORD}
```

Comment:
probably can leave this and the det deploy stuff out for now. is it just me or does it looks like there's not even a way to supply them through the cli?

Reply:
will do. there's no way to supply them here, and ee master doesn't consume these either.